wandb/integration/langchain/util.py

```diff
@@ -94,92 +92,78 @@ def safely_convert_model_to_dict(
     return data
 
 
-def _convert_lc_run_to_wb_span(run: "BaseRun") -> "trace_tree.Span":
-    if isinstance(run, LLMRun):
+def _convert_lc_run_to_wb_span(run: "Run") -> "trace_tree.Span":
+    if run.run_type == RunTypeEnum.llm:
         return _convert_llm_run_to_wb_span(run)
-    elif isinstance(run, ChainRun):
+    elif run.run_type == RunTypeEnum.chain:
         return _convert_chain_run_to_wb_span(run)
-    elif isinstance(run, ToolRun):
+    elif run.run_type == RunTypeEnum.tool:
         return _convert_tool_run_to_wb_span(run)
     else:
         return _convert_run_to_wb_span(run)
 
 
-def _convert_llm_run_to_wb_span(run: "LLMRun") -> "trace_tree.Span":
+def _convert_llm_run_to_wb_span(run: "Run") -> "trace_tree.Span":
     base_span = _convert_run_to_wb_span(run)
 
-    if run.response is not None:
-        base_span.attributes["llm_output"] = run.response.llm_output
+    if run.outputs is not None:
+        base_span.attributes["llm_output"] = run.outputs
```

Comment:
We can just remove this section - run.outputs is captured below and `llm_outputs` looks like it was removed from langchain